Modules/clinic/posixmodule.c.h

```diff
@@ -1849,7 +1849,8 @@ PyDoc_STRVAR(os_mkdir__doc__,
 "dir_fd may not be implemented on your platform.\n"
 "  If it is unavailable, using it will raise a NotImplementedError.\n"
 "\n"
-"The mode argument is ignored on Windows.");
+"The mode argument is ignored on Windows. Where it is used, the current umask\n"
```

Comment:
This is editing the generated output of the "argument clinic" program.  What you need to do is edit the [source file here](https://github.com/python/cpython/blob/main/Modules/posixmodule.c#L4524), run `make clinic` to regenerate the output, and commit both changes to your PR.

Reply:
Oh, thank you very much! Done.